server/db/models/quantityhavingorder.js

```diff
@@ -1,7 +1,7 @@
 const Sequelize = require('sequelize');
 const db = require('../db');
 
-const ProductOrderedQuantity = db.define('productOrderedQuantity', {
+const QuantityHavingOrder = db.define('quantityhavingorder', {
```

Comment:
I'm not too sure about the name for this model...but I don't have a better one to suggest! Maybe we can check with the whole team and see if we can find a better one?